src/leap/mail/incoming/service.py

```diff
@@ -729,7 +729,7 @@ def signal_deleted(doc_id):
             d.addCallback(signal_deleted)
             return d
 
-        d = self._inbox.add_raw_message(data, (self.RECENT_FLAG,))
+        d = self._inbox.add_msg(data, (self.RECENT_FLAG,))
```

Comment:
right, you're right.
the internal date is not the date in the headers, according to specs this has different meanings.
I've clarified the docstring, and also I'm adding the insertion date from the incoming service now.